paasta_tools/secret_providers/vault.py

```diff
@@ -11,7 +11,7 @@
     from vault_tools.paasta_secret import get_vault_client
     from vault_tools.gpg import TempGpgKeyring
     from vault_tools.paasta_secret import encrypt_secret
-    from vault_tools.cert_tools import do_renew
+    from vault_tools.cert_tools import do_cert_renew
     import hvac
 except ImportError:
```

Comment:
tests didn't fail because of this. do we really want to ignore import errors? if this is necessary lets add some constant that indicates whether modules were loaded or not and fail tests on the valuer of that constant.

Reply:
`vault-tools` is an internal library which will never be loaded in tests (which run publicly), so we have no real way of checking that.